test/gui/shared/scripts/helpers/api/oc10.py

```diff
@@ -0,0 +1,58 @@
+import json
+
+import helpers.api.HttpHelper as request
+from helpers.api.utils import url_join
+from helpers.ConfigHelper import get_config
+
+created_groups = {}
+
+
+def get_ocs_url():
+    return url_join(get_config('localBackendUrl'), 'ocs', "v2.php", 'cloud')
+
+
+def checkSuccessOcsStatus(response):
+    if response.text:
+        ocs_code = json.loads(response.text)['ocs']['meta']['statuscode']
+        if ocs_code not in [100, 200]:
+            raise Exception("Request failed." + response.text)
+    else:
+        raise Exception(
+            "No OCS response body. HTTP status was " + str(response.status_code)
+        )
+
+
+def create_group_ocs(group_name):
+    url = url_join(get_ocs_url(), 'groups')
+    body = {"groupid": group_name}
+    response = request.post(url, body)
+    if response.status_code != 200:
+        raise Exception(
+            "Creating group '%s' failed with %s\n" % (group_name, response.status_code)
+            + response.text
+        )
```

Comment:
could have be assertion function in separate file if need to reuse anywhere
```suggestion
    assertHttpStatus(response, 200)
```